docs/platform/contracts/broadcast.md

```diff
@@ -0,0 +1,12 @@
+**Usage**: `sdk.platform.contracts.broadcast(contract, identity)`    
+**Description**: This method will create an `application` or `user` new identity for you. 
```

Comment:
Copy/paste error? Looks like this was not updated for contracts.

Reply:
It is, I create files by templating the previous one. I forgot about this. Fixing it, thanks !

Reply:
Fixed thank you.